rise-storage.html

```diff
@@ -215,9 +224,9 @@
      *
      * @property _baseCacheUrl
      * @type string
-     * @default "//localhost:9494/"
+     * @default "//localhost:9494/files/"
      */
-    _baseCacheUrl: "//localhost:9494/",
+    _baseCacheUrl: "//localhost:9494/files/",
```

Comment:
@donnapep I think the base url should be just localhost:9494 as we also have the metadata endpoint. cheers